tests/integration/tier0/non-destructive/grub/test_invalid_changed_grub.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import print_function
+
+import fileinput
+import os.path
+
+import pytest
+
+from conftest import TEST_VARS
+
+
+target_line = "GRUB_CMDLINE_LINUX"
+
+
+@pytest.fixture
+def grub_file_invalid(shell_fixture, backup_directory):
```

Comment:
why `shell_fixture` instead of just `shell`?

Reply:
That might be a result of auto-completion and laziness :no_mouth: 
Good catch though